qunicorn_core/core/pilotmanager/ibm_pilot.py

```diff
@@ -292,7 +292,7 @@ def __map_runner_results_to_dataclass(
             metadata["registers"] = classical_registers_metadata
             data: Dict = metadata.pop("data")
             metadata.pop("circuit", None)
-            # probabilities: dict = Pilot.calculate_probabilities(counts)  # TODO: append as extra result
+            probabilities: dict = Pilot.calculate_probabilities(data.get("counts", {}))
```

Comment:
Do we want to include result formats by default that require extra computation on our end?